public/javascripts/Gallery/src/displays/ValidationInfoDisplay.js

```diff
@@ -5,10 +5,13 @@
  * @param {HTMLElement} container The DOM element that contains the display
  * @param {Number} agreeCount The agree count to display
  * @param {Number} disagreeCount The disagree count to display
- * @param {Boolean} isModal a toggle to determine if this SeverityDisplay is in a modal, or in a card
+ * @param {Boolean} isExpandedView a toggle to determine if this ValidationInfoDisplay is in expanded view, or in a card
  * @returns {ValidationInfoDisplay} the generated object
  */
-function ValidationInfoDisplay(container, agreeCount, disagreeCount, isModal=false) {
+function ValidationInfoDisplay(container,
```

Comment:
Our default is to keep these param lists on one line, let's just keep it that way for now. Can see an argument for changing that in the future, but let's just stay consistent for now!